template.yaml

```diff
@@ -6,18 +6,27 @@ Description: >
   SAM template for notebook data redirector
 
 Parameters:
-  ManifestTableName:
```

Comment:
Turns out we can have CloudFormation generate a table name based on the stack name